packages/web-extension/test/walletManager/WalletRepository.test.ts

```diff
@@ -344,6 +424,25 @@ describe('WalletRepository', () => {
       ]);
     });
 
+    it('does not remove 1852 account when removing 1854 account', async () => {
+      const newAccount = createAccount(0, 1, KeyPurpose.MULTI_SIG);
+      const accounts = [storedLedgerWallet.accounts[0], newAccount];
```

Comment:
same for `does not update 1852 account metadata when updating 1854 account`
```suggestion
      const storedAccount = storedLedgerWallet.accounts[0];
      const accounts = [storedAccount, newAccount];
```

Reply:
Fixed on 19c238b